docs(Configuration): add output.workerPublicPath (#6712)
webpack/webpack.js.org: src/content/configuration/output.mdx

  },
};
```

## output.workerPublicPath

`string`

> Set a public path for Worker, defaults to value of [output.publicPath](#outputpublicpath). Avoid this option unless your worker scripts are located at a different path from your other script files.

Author: ```suggestion
Set a public path for Worker, defaults to value of [output.publicPath](#outputpublicpath). Only use this option if your worker scripts are located in a different path from your other scripts.
```
